libpod/image/image.go

```diff
@@ -156,6 +156,41 @@ func (ir *Runtime) New(ctx context.Context, name, signaturePolicyPath, authfile
 	return &newImage, nil
 }
 
+// NewFromLoad creates a new image object for images pulled from a tar archive (podman load)
+// This function is needed because it is possible for a tar archive to have multiple tags for one image
+func (ir *Runtime) NewFromLoad(ctx context.Context, name, signaturePolicyPath string, writer io.Writer) ([]*Image, error) {
+	var newImages []*Image
+	newImage := Image{
+		InputName:    name,
+		Local:        false,
+		imageruntime: ir,
+	}
+
+	if signaturePolicyPath == "" {
+		signaturePolicyPath = ir.SignaturePolicyPath
+	}
+	imageNames, err := newImage.pullImage(ctx, writer, "", signaturePolicyPath, SigningOptions{}, &DockerRegistryOptions{}, false)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to pull %s", name)
+	}
+
+	for _, name := range imageNames {
+		newImage := Image{
+			InputName:    name,
+			Local:        false,
```

Comment:
Shouldn't local be true, given that we have pulled the image?

Reply:
The image is in the archive file, if that is considered Local, I can set that true. I thought local meant it was in the container storage.

Reply:
Hasn't it already been loaded into c/storage at this point via the pull in here?

Reply:
Oh right, changing that.

Reply:
fixed